Extensions/Steamworks/Z_steamworksinputtools.ts

```diff
@@ -13,53 +13,51 @@ namespace gdjs {
     export function activateActionSet(
       controllerIndex: number,
       actionSetName: string
-    ) {
-      if (gdjs.steamworks.steamAPI) {
-        gdjs.steamworks.steamAPI.input
-          .getControllers()
-          [controllerIndex]?.activateActionSet(
-            gdjs.steamworks.steamAPI.input.getActionSet(actionSetName)
-          );
-      }
+    ): void {
+      if (!gdjs.steamworks.steamAPI) return;
+      gdjs.steamworks.steamAPI.input
+        .getControllers()
+        [controllerIndex]?.activateActionSet(
+          gdjs.steamworks.steamAPI.input.getActionSet(actionSetName)
+        );
     }
 
     export function isDigitalActionPressed(
       controllerIndex: number,
       actionName: string
-    ) {
-      if (gdjs.steamworks.steamAPI) {
-        gdjs.steamworks.steamAPI.input
-          .getControllers()
-          [controllerIndex]?.isDigitalActionPressed(
-            gdjs.steamworks.steamAPI.input.getDigitalAction(actionName)
-          );
-      }
+    ): boolean {
+      if (!gdjs.steamworks.steamAPI) return false;
+      return gdjs.steamworks.steamAPI.input
+        .getControllers()
+        [controllerIndex]?.isDigitalActionPressed(
+          gdjs.steamworks.steamAPI.input.getDigitalAction(actionName)
+        );
     }
 
     export function getAnalogActionVectorX(
       controllerIndex: number,
       actionName: string
-    ) {
-      if (gdjs.steamworks.steamAPI) {
-        gdjs.steamworks.steamAPI.input
-          .getControllers()
-          [controllerIndex]?.getAnalogActionVector(
-            gdjs.steamworks.steamAPI.input.getAnalogAction(actionName)
-          ).x;
-      }
+    ): float {
+      return gdjs.steamworks.steamAPI
+        ? gdjs.steamworks.steamAPI.input
+            .getControllers()
+            [controllerIndex]?.getAnalogActionVector(
+              gdjs.steamworks.steamAPI.input.getAnalogAction(actionName)
+            ).x ?? 0
```

Comment:
We have not used typescript operators anywhere I think, I guess you could use `||` instead?

Reply:
The nullish coalescing operator is not a TypeScript operator, it is [part of the ECMAScript standard specification](https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-CoalesceExpression) and has been [supported by all browsers (except IE) for a while now](https://caniuse.com/mdn-javascript_operators_nullish_coalescing).